### ISSUE TITLE
Cannot set exact headers

### ISSUE THREAD

**Issue by plokhotnyuk** [outr/youi#79]
I need to set the following headers:
```
< Server: youi
< Content-Type: application/json
< Content-Length: 27
< Date: Sat, 02 Feb 2019 13:11:50 GMT
```

But youi server send response with these headers:
```
< Connection: keep-alive
< Transfer-Encoding: chunked
< Date: Sat, 02 Feb 2019 13:14:17 GMT
```

How it can be coded for the most efficiency in runtime? 
A branch with the sample code is [here](https://github.com/plokhotnyuk/FrameworkBenchmarks/blob/add-youi-server/frameworks/Scala/youi/src/main/scala/example/Main.scala#L14).

**Comment by darkfrog26** [comment]
Publishing `0.10.2` right now that will fix all of this. The `0.10` release introduced a bug that was keeping the content-type from being set automatically like it's supposed to.

In addition, if you want to set custom headers you can always do:
```scala
handlers += new HttpHandler {
      override def handle(connection: HttpConnection): Future[HttpConnection] = {
        Future.successful(connection.modify { response =>
          response.withHeader("Server", "youi")
        })
      }
    }
```

However, I've also introduced a `Server` header that defaults to "youi" as well, so this should all be resolved with `0.10.2` automatically.

**Comment by darkfrog26** [comment]
Okay, publish is complete. It's available on Sonatype, but it may take a while before it's synchronized to Maven Central.
